pkg/client/state_changes.go

```diff
@@ -21,8 +21,10 @@ type IssueAction struct {
 	Name           string        `json:"name"`
 	Description    string        `json:"description"`
 	Decimals       int32         `json:"decimals"`
+	Quantity       int64         `json:"quantity"`
 	Reissuable     bool          `json:"isReissuable"`
-	CompiledScript []byte        `json:"compiledScript"`
+	CompiledScript *[]byte       `json:"compiledScript"` // optional
```

Comment:
Pointer here is unnecessary because zero value of any slice is `nil`.

Reply:
Oh, I've realized, that yes. Closing.

Reply:
done